src/model.rs

```diff
@@ -166,6 +184,78 @@ impl PodJob {
     }
 }
 
+fn serialize_pod_job<S>(pod_job: &PodJob, serializer: S) -> result::Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    serializer.serialize_str(&pod_job.hash)
+}
+
+fn deserialize_pod_job<'de, D>(deserializer: D) -> result::Result<PodJob, D::Error>
```

Comment:
Same thing as pod, I recall skip with default would allow this behavior to be done